internal/pkg/api/handlers/auth_test.go

```
@@ -307,4 +308,176 @@ func TestGetCookieConfig(t *testing.T) {
+			"name":                       mockUser.Name,
+			"profile_picture_url":        mockUser.ProfilePictureURL,
+			"timezone":                   mockUser.Timezone,
+			"email_notifications_enabled": mockUser.EmailNotificationsEnabled,
+			"automation_enabled":         mockUser.AutomationEnabled,
+			"has_strava_connection":      mockUser.HasStravaConnection,
+			"has_sheets_connection":      mockUser.HasSheetsConnection,
+			"recent_activity_logs":       []interface{}{}, // Empty array for now
+		}
+
+		w.Header().Set("Content-Type", "application/json")
+		json.NewEncoder(w).Encode(response)
```

golangci-lint (1.64.8)

Error return value of `(*encoding/json.Encoder).Encode` is not checked

(errcheck)